platform/platform-filter-delegate/src/main/java/org/codice/ddf/platform/filter/delegate/FilterInjector.java

```diff
@@ -173,4 +187,45 @@ public void injectFilter(ServiceReference<ServletContext> serviceReference) {
         }
     }
 
+    public void addListener(ServiceReference<HttpSessionListener> sessionListener) {
+        if (sessionListener != null) {
+            Bundle refBundle = sessionListener.getBundle();
+            if (refBundle != null) {
+                BundleContext bundlectx = refBundle.getBundleContext();
+                if (bundlectx != null) {
+                    sessionListeners.add(bundlectx.getService(sessionListener));
```

Comment:
1. Times like this, I really miss Groovy and its safe navigation operator.
2. Should probably check that you got a service back so as not to add a null to the list.